app/components/HomePage/ActionsSection/ActionsSection.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { scrollTrigger } from 'utils/animation';
+import screen_1 from 'images/scr_1.jpg';
+import screen_2 from 'images/scr_2.jpg';
+import screen_3 from 'images/scr_3.jpg';
+import screen_4 from 'images/scr_4.jpg';
+import messages from '../messages';
+
+import { styles } from './ActionsSection.styled';
+import { pageStyles } from '../HomePage.styled';
+
+const ActionsSection: React.FC = (): JSX.Element => {
+  const [startFirstActionAnimation, setStartFirstActionAnimation] =
+    useState<boolean>(false);
```

Comment:
why variables to start animation?

you can use component
```
import { CSSProperties } from 'react'

import { useInView } from 'react-intersection-observer'

type Props = {
  style?: CSSProperties
  className?: string
}

const InViewport: React.FC<Props> = ({
  children,
  style,
  className,
}): JSX.Element => {
  const { ref, inView } = useInView({ triggerOnce: true })
  return (
    <div className={className} style={style} ref={ref}>
      {inView && children}
    </div>
  )
}

export default InViewport
```

Reply:
-